items.h

```diff
@@ -96,7 +96,8 @@ class ItemType
 			// This item cannot be transported.
 			NOTRANSPORT = 0x20,
 			// Produced monsters
-			MANPRODUCE = 0x40
+			MANPRODUCE = 0x40,
+			SKILLOUT_PROGRESSIVE = 0x50
```

Comment:
We have similar feature in other skills and it is named differently. From the name "progressive" is not clear what it does.

Reply:
This is an output multiplier. Maybe use flag name that reflects such behaviour?

Reply:
sure, I will change name